packages/documentation/src/stories/internet-header/components/header/header.stories.ts

```diff
@@ -217,3 +215,14 @@ export const OSFlyoutOverrides: Story = {
     },
   },
 };
+
+export const CustomContent: Story = {
+  args: {
+    innerHTML: `<p class="mx-regular mb-0" slot="main">Hello, User!</p>`,
+  },
+  parameters: {
+    controls: {
+      include: ['innerHTML'],
+    },
+  },
```

Comment:
I tried adding custom content a bit, re-styling the `<p>` etc. and the preview breaks fairly often, because while writing, the HTML is invalid and starts to throw errors. Also copy/pasting does not work always.

I'm not sure if we should actually show this control, in this state it adds little value and fixing it might be a rabbit hole.